libpod/runtime.go

```diff
@@ -398,9 +398,13 @@ func makeRuntime(runtime *Runtime) (err error) {
 	}
 
 	// Set up containers/storage
-	store, err := storage.GetStore(runtime.config.StorageConfig)
-	if err != nil {
-		return err
+	var store storage.Store
+	if !rootless.SkipStorageSetup() {
+		logrus.Debug("Not configuring container store")
```

Comment:
Can you move this into an else so it only prints if we're not configuring the store?